cmd/utils/flags.go

```diff
@@ -144,6 +144,11 @@ var (
 		Usage: "Data directory for the databases and keystore",
 		Value: DirectoryString(node.DefaultDataDir()),
 	}
+	RaftLogDirFlag = DirectoryFlag{
+		Name:  "raftlogdir",
+		Usage: "Raft log directory for the raft-state, raft-snap and raft-wal folders",
+		Value: DirectoryString(node.DefaultDataDir()),
```

Comment:
This sets the default directory to be the same location as the default data directory. If the `datadir` changes, this won't update with it. Keeping that assumption the same, particularly for existing nodes, is important.

I added a possible solution in the other comments.